web/src/engine/osk/src/banner/suggestionBanner.ts

```diff
@@ -79,28 +145,162 @@ export class BannerSuggestion {
    * Function update
    * @param {string}     id           Element ID for the suggestion span
    * @param {Suggestion} suggestion   Suggestion from the lexical model
+   * @param fontStyle                 The CSS styling expected for the suggestion text
+   * @param emSize                    The font size represented by 1em (in px, as from getComputedStyle on document.body)
+   * @param targetWidth
+   * @param collapsedTargetWidth
```

Comment:
Technically, these four are part of `format`. Could do `@param format.fontStyle` etc.